packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -271,10 +272,11 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
         const request = { url: blobId };
         if (this.tombstonedBlobs.has(blobId) ) {
             const error = responseToException(createResponseError(404, "Blob removed by gc", request), request);
-            this.mc.logger.sendErrorEvent({
+            const event = {
                 eventName: "GC_Tombstone_Blob_Requested",
                 url: request.url,
-            }, error);
+            };
+            sendGCTombstoneEvent(this.mc, event, this.runtime.clientDetails.type === summarizerClientType, [BlobManager.basePath], error);
```

Comment:
You should add isSummarizerClient and packagePath to the event. Or, you should pass all properties directly to the function. It seems inconsistent currently.

Reply:
I did this so that all calls had to add this to the event.